test/simple-concat-test.js

```diff
@@ -504,7 +505,7 @@ describe('simple-concat', function() {
         fs.removeSync(dirPath);
         inputNodesOutput = [];
 
-        node = concat(firstFixture, {
+        node = concat(new UnwatchedDir(firstFixture), {
```

Comment:
This was necessary to uncover the misbehavior in tests, as `broccoli-debug` would only omit creating a debug node when the passed input is a object (real broccoli node), not just a string.